common/src/main/java/org/apache/celeborn/common/protocol/message/StatusCode.java

```diff
@@ -58,24 +58,26 @@ public enum StatusCode {
   COMMIT_FILE_EXCEPTION(29),
 
   // Rate limit statuses
-  PUSH_DATA_SUCCESS_MASTER_CONGESTED(30),
-  PUSH_DATA_SUCCESS_SLAVE_CONGESTED(31),
+  PUSH_DATA_SUCCESS_PRIMARY_CONGESTED(30),
+  PUSH_DATA_SUCCESS_REPLICA_CONGESTED(31),
 
-  PUSH_DATA_HANDSHAKE_FAIL_SLAVE(32),
-  PUSH_DATA_HANDSHAKE_FAIL_MASTER(33),
-  REGION_START_FAIL_SLAVE(34),
-  REGION_START_FAIL_MASTER(35),
-  REGION_FINISH_FAIL_SLAVE(36),
-  REGION_FINISH_FAIL_MASTER(37),
+  PUSH_DATA_HANDSHAKE_FAIL_REPLICA(32),
+  PUSH_DATA_HANDSHAKE_FAIL_PRIMARY(33),
+  REGION_START_FAIL_REPLICA(34),
+  REGION_START_FAIL_PRIMARY(35),
+  REGION_FINISH_FAIL_REPLICA(36),
+  REGION_FINISH_FAIL_PRIMARY(37),
 
-  PUSH_DATA_CREATE_CONNECTION_FAIL_MASTER(38),
-  PUSH_DATA_CREATE_CONNECTION_FAIL_SLAVE(39),
-  PUSH_DATA_CONNECTION_EXCEPTION_MASTER(40),
-  PUSH_DATA_CONNECTION_EXCEPTION_SLAVE(41),
-  PUSH_DATA_TIMEOUT_MASTER(42),
-  PUSH_DATA_TIMEOUT_SLAVE(43),
-  PUSH_DATA_MASTER_WORKER_EXCLUDED(44),
-  PUSH_DATA_SLAVE_WORKER_EXCLUDED(45),
+  PUSH_DATA_CREATE_CONNECTION_FAIL_PRIMARY(38),
```

Comment:
Change this will make blacklist won't work during upgrade, but I think it's ok cc @waitinfuture @pan3793

Reply:
I think it's fine for 0.2, but do we have general solutions for future breaking change?